bench.c

```diff
@@ -83,12 +83,16 @@ static void *bench_worker(__attribute__((unused)))
 
     /* wait until all workers created */
     pthread_mutex_lock(&worker_lock);
-    while (!ready)
-        if (pthread_cond_wait(&worker_wait, &worker_lock)) {
-            puts("pthread_cond_wait failed");
-            exit(-1);
-        }
+    if (++ready == MAX_THREAD)
```

Comment:
@kevinshieh0225 , Is `ready` a shared variable between threads? Do we need atomic addition?

Reply:
Yes, but I think the variable addition is protected by the mutex lock.